ui/src/store/api.ts

```diff
@@ -0,0 +1,69 @@
+// Package import
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+// Constant import
+import constants from '../app.constants';
+
+const api = createApi({
```

Comment:
Add a comment explaining what are you doing in blow code.

Reply:
Done